client/src/components/Logout.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+/* eslint-disable require-jsdoc, class-methods-use-this */
+export default class Logout extends React.Component {
+  render() {
```

Comment:
Since this is a stateless functional component what you want to do is
```
export default () => {
  return (
    // div
  );
}
```

Reply:
Thank you for the feedback